docs/semgrep-secrets/historical-scanning.md

```diff
@@ -0,0 +1,119 @@
+
+### Prerequisites
+
+- **CLI tool**: Historical scanning requires at least Semgrep **v1.65.0**. See [Update](/update/) for instructions.
+- **Single-tenant Semgrep Cloud Platform (SCP)**: Reach out to your TAM to ensure your instance is up-to-date.
+
+### Enable historical scanning for full Secrets scans
+
+:::tip
+If possible, [test historical scans locally](#run-a-local-test-scan) to create a benchmark of performance and scan times before adding historical scans to your formal security process.
+:::
+
+1. Sign in to Semgrep Cloud Platform.
+1. Click **<i class="fa-solid fa-gear"></i> Settings**.
+1. Under Deployment, click the **<i class="fa-solid fa-toggle-large-on"></i> Historical scanning** toggle.
+![Historical scanning settings toggle](/img/historical-scanning-settings.png#md-width)
+
+Subsequent Semgrep full scans now include historical scanning.
+
+### Run a one-off historical scan
+
+To run a one-off or on-demand historical scan, you can create a specific CI job and then manually start the job as needed.
+
+The general steps are:
+
+1. Copy your current full scan CI job configuration file, or use [a template](/semgrep-ci/sample-ci-configs/).
+1. Look for the `semgrep ci` command.
+1. Append the `--historical-secrets` flag:
+    ```
+    semgrep ci --historical-secrets
+    ```
+1. Depending on your CI provider, you may have to perform additional steps to enable the job to run manually. For example, GitHub Actions requires the `workflow_dispatch` event to be added to your CI job.
+
+### Run a local test scan
+
+You can run a historical scan locally without sending the scan results to Semgrep Cloud Platform. This can help you determine the time it takes for Semgrep Secrets to run on your repository's Git commit history.
+
+To run a test scan, enter the following command:
+
+```bash
+semgrep scan --historical-secrets
+```
+
+The historical scan results appear in the **Secrets Historical Scan** section:
+
+![Historical scan section in the CLI](/img/historical-scans-cli.png#md-width)
+
+## View or hide historical findings
+
+1. Sign in to [<i class="fas fa-external-link fa-xs"></i> Semgrep Cloud Platform](https://semgrep.dev/login).
+1. Click **<i class="fa-solid fa-key"></i> Secrets**. Historical findings are identified by an **<i class="fa-solid fa-hourglass-half"></i>** icon.
+1. Click **<i class="fa-solid fa-hourglass-half"></i> Hide historical** to toggle the display of historical findings.
+
+![Historical secrets in Semgrep Cloud Platform](/img/historical-secrets-scp.png)
+*Figure.* Historical findings in Semgrep Cloud Platform.
+
+## Scope of findings
+
+- Historical scans display **valid** Secrets findings. These secrets have been validated, through authentication or a similar function, to grant access to their service or site.
```

Comment:
This bullet is kinda wordy, we could also reference our existing docs instead of re-explaining some of this stuff if you wanted


https://semgrep.dev/docs/semgrep-secrets/conceptual-overview/#validate-secrets

Reply:
Thanks. Linked it 2x to make sure people find it. 

![Snag_1500627](https://github.com/semgrep/semgrep-docs/assets/20766840/4ffa39fc-7b7a-42b4-8723-3dbf39fb26fe)